AirSkyBoat/AirSkyBoat
# Add Mount Zhayolm gates and cast metal plate functionality

**_I affirm:_**


## Please enter a player-facing description
none (for ToAU launch)

## What does this pull request do? (Please be technical)
This adds mount zhayolm gates and cast metal plate functionality that is currently missing from LSB

## Steps to test these changes
1. Go to cast metal plate (!gotoid 17027570) and try to get KI without touching a gate beforehand
2. Go to a gate (!pos 189.000 -19.000 -20.000) and click on it (should not open)
3. Go again to cast metal plate (!gotoid 17027570) and try to get KI (should get the KI now)
4. ??? should have moved, go again to cast metal plate (!gotoid 17027570) and click and it should say nothing out of ordinary
5. Go to a gate (!pos 189.000 -19.000 -20.000) and click on it (should now open)

## Special Deployment Considerations
